tests/DqtApi.Tests/V2/Operations/UpdateTeacherTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using DqtApi.DataStore.Crm.Models;
+using DqtApi.TestCommon;
+using DqtApi.V2.ApiModels;
+using DqtApi.V2.Requests;
+using Microsoft.AspNetCore.Http;
+using Moq;
+using Xunit;
+
+namespace DqtApi.Tests.V2.Operations
+{
+    public class UpdateTeacherTests : ApiTestBase
```

Comment:
Would be good to have test for the success case and also various error cases like https://github.com/DFE-Digital/qualified-teachers-api/blob/main/tests/DqtApi.Tests/V2/Operations/GetOrCreateTrnRequestTests.cs#L161-L324